### PR TITLE
Bugfix/bsc#1051838

### PR DESCRIPTION
**Pull request by lproven**
In a comment on Bugzilla, Thorsten Kukuk asked for an example using a password hash rather than plaintext.

This also makes the separate worker-node and admin-node example files more useful.

### CODE REVIEW

File: xml/deployment_installation.xml [hunk 1]
```diff
@@ -861,21 +861,19 @@
...
        So, for instance, if your working directory is <filename>~/cloud-config</filename>,
-       then for the admin node, you need:
+       then for the admin node, you need these two files:
       </para>
-      <screen>~/cloud-config/cc-admin/user-data</screen>
-      <para>
-       and
-      </para>
-      <screen>~/cloud-config/cc-admin/meta-data</screen>
+      <screen>
```
Comment (cwickert) [inline xml/deployment_installation.xml]: no line break

File: xml/deployment_installation.xml [hunk 2]
```diff
@@ -861,21 +861,19 @@
...
+~/cloud-config/cc-admin/user-data
+~/cloud-config/cc-admin/meta-data
```
Comment (cwickert) [inline xml/deployment_installation.xml]: no line break

File: xml/deployment_installation.xml [hunk 3]
```diff
@@ -861,21 +861,19 @@
...
+      </screen>
       <para>
        For a &worker_node;, you need:
       </para>
-      <screen>~/cloud-config/cc-worker/user-data</screen>
-      <para>
-       and
-      </para>
-      <screen>~/cloud-config/cc-worker/meta-data</screen>
+      <screen>
```
Comment (cwickert) [inline xml/deployment_installation.xml]: no line break

File: xml/deployment_installation.xml [hunk 4]
```diff
@@ -861,21 +861,19 @@
...
+~/cloud-config/cc-worker/user-data
+~/cloud-config/cc-worker/meta-data
```
Comment (cwickert) [inline xml/deployment_installation.xml]: no line break

File: xml/deployment_installation.xml [hunk 5]
```diff
@@ -899,10 +897,11 @@ network-interfaces: |
 #cloud-config
 debug: True
 disable_root: False
+ssh_deletekeys: False
```
Comment (cwickert) [inline xml/deployment_installation.xml]: Care to elaborate?

File: xml/deployment_installation.xml [hunk 6]
```diff
@@ -899,10 +897,11 @@ network-interfaces: |
...
 ssh_pwauth: True
 chpasswd:
   list: |
-    root:<replaceable>root</replaceable>
+    root:<replaceable>MyPassword</replaceable>
```
Comment (cwickert) [inline xml/deployment_installation.xml]: MY_PASSWORD (replacables are written in caps)

File: xml/deployment_installation.xml [hunk 7]
```diff
@@ -921,10 +920,11 @@ suse_caasp:
...
```
Comment (cwickert) [inline xml/deployment_installation.xml]: MY_PASSWORD

File: xml/deployment_installation.xml [hunk 8]
```diff
@@ -936,20 +936,28 @@ suse_caasp:
...
       </screen>
       <para>
-       To create the ISO image for a worker node, use the same command, but 
-       substitute the name of the folder containing the configuration files for
-       a &worker_node; and title the volume <literal>cc-worker</literal>.
+       To create the ISO image for a worker node on a computer running &opensuse;
+       15, substituting the name of the folder containing the configuration
+       files for a &worker_node; and titling the volume <literal>cc-worker</literal>:
       </para>
+      <screen>
```
Comment (cwickert) [inline xml/deployment_installation.xml]: no line break

File: xml/deployment_installation.xml [hunk 9]
```diff
@@ -936,20 +936,28 @@ suse_caasp:
...
+&prompt.user; sudo mkisofs -output <replaceable>cc-worker.img</replaceable> -volid <replaceable>cidata</replaceable> -joliet -rock <replaceable>cc-worker</replaceable>
```
Comment (cwickert) [inline xml/deployment_installation.xml]: no whitespace after &prompt.user, it's already part of the entity.
tag command as <command>.
wrap the command line with \ if it doesn't fit into a single line in the pdf version.
no line break before the end of the screen

File: xml/deployment_installation.xml (hunk 1 above)
Comment (lproven) [reply xml/deployment_installation.xml]: Done.

File: xml/deployment_installation.xml (hunk 2 above)
Comment (lproven) [reply xml/deployment_installation.xml]: Done.

File: xml/deployment_installation.xml (hunk 3 above)
Comment (lproven) [reply xml/deployment_installation.xml]: Done.

File: xml/deployment_installation.xml (hunk 6 above)
Comment (lproven) [reply xml/deployment_installation.xml]: Done.

File: xml/deployment_installation.xml (hunk 7 above)
Comment (lproven) [reply xml/deployment_installation.xml]: I can only find one instance of "MyPassword".

File: xml/deployment_installation.xml (hunk 5 above)
Comment (lproven) [reply xml/deployment_installation.xml]: No -- I was taking the example files from Bugzilla. 

Googling suggests that "for public instances this should definitely be 'true'". Happy to change it if you think that is preferable.

**Comment by lproven** [comment]
I think I have addressed all your comments now. If there are any outstanding, please tell me.

The build failure seems to be due to some external SLE material but I am not experienced at interpreting Travis output.

File: xml/deployment_installation.xml (hunk 6 above)
Comment (cwickert) [reply xml/deployment_installation.xml]: The team hasn't agreed on a solution yet.

Comment (lproven) [reply xml/deployment_installation.xml]: OK, fair enough. Are you happy with my amendments to this at present, though, or do you want more changes?

**Pull request merged**